2048/2048/src/main/java/com/appcoins/eskills2048/LaunchActivity.java

```diff
@@ -34,13 +34,16 @@
 
   public static final String USER_ID = "USER_ID";
   public static final String WALLET_ADDRESS = "WALLET_ADDRESS";
+  public static final String SKU = "SKU";
+  public static final String MATCH_ENVIRONMENT = "MATCH_ENVIRONMENT";
   public static final String SESSION = "SESSION";
   public static final String LOCAL_GAME_STATUS = "LOCAL_GAME_STATUS";
 
   private static final String ENTRY_PRICE_DUEL = "1 USD";
   private static final String ENTRY_PRICE_MULTIPLAYER = "4 USD";
 
   private final String userId = "string_user_id";
+  private MatchDetails.Environment match_environment;
```

Comment:
for naming variables in java or kotlin, you should use camelCase. you can follow this document regarding variables by oracle: https://docs.oracle.com/javase/tutorial/java/nutsandbolts/variables.html

Reply:
fixed